### DISCUSSION TITLE
digitalpalireader.online is down, working on bringing it up

### DISCUSSION THREAD

**Discussion by parthopdas** [kitamstudios/digitalpalireader#397] [General]
hello, digitalpalireader.online is down, i am working on bringing it up. ETA 1 day as i need help from azure support.

if you DPR downloaded offline this should not affect you.

**Comment by thesunshade** [comment]
Much gratitude for your work!

**Comment by parthopdas** [comment]
fyi dpr is backonline. both https://digitalpalireader.online/  and https://staging.digitalpalireader.online/

please report any issues here.

**Reply by thesunshade** [reply]
Thank you!
